apps/andi/lib/andi_web/input_schemas/metadata.ex

```diff
@@ -0,0 +1,109 @@
+defmodule AndiWeb.InputSchemas.Metadata do
+  @moduledoc false
+  import Ecto.Changeset
+
+  @business_fields %{
+    contactEmail: :string,
+    contactName: :string,
+    dataTitle: :string,
+    description: :string,
+    homepage: :string,
+    issuedDate: :date,
+    keywords: {:array, :string},
+    language: :string,
+    license: :string,
+    modifiedDate: :date,
+    orgTitle: :string,
+    publishFrequency: :string,
+    spatial: :string,
+    temporal: :string
+  }
+
+  @technical_fields %{
+    private: :boolean,
+    sourceFormat: :string
+  }
+
+  @types Map.merge(@business_fields, @technical_fields)
+
+  @email_regex ~r/^[A-Za-z0-9._%+-+']+@[A-Za-z0-9.-]+\.[A-Za-z]+$/
```

Comment:
This looks like it's outside the scope of your card. Also, I don't see it being used anywhere.

Reply:
We just moved it out from the place